airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -525,16 +527,13 @@ def __init__(self, parent: HttpStream, **kwargs: Any):
     def stream_slices(
         self, sync_mode: SyncMode, cursor_field: Optional[List[str]] = None, stream_state: Optional[Mapping[str, Any]] = None
     ) -> Iterable[Optional[Mapping[str, Any]]]:
-        parent_stream_slices = self.parent.stream_slices(
-            sync_mode=SyncMode.full_refresh, cursor_field=cursor_field, stream_state=stream_state
-        )
-
-        # iterate over all parent stream_slices
-        for stream_slice in parent_stream_slices:
```

Comment:
I still see cases where some sources use `stream_slices` and `read_record` afterward (like [this](https://github.com/airbytehq/airbyte/blob/1d125599db9c6bfcd15e700dfb9ca0190dc3ecd4/airbyte-integrations/connectors/source-stripe/source_stripe/streams.py#L524-L526)). Are we fine with those because they don't rely on low-code so they shouldn't be affected by the RFR cursor?

Reply:
Good point and that's correct. This bug was only for low-code because all non-incremental and non-substreams for low-code automatically turned on RFR. whereas for python based sources streams are currently opt in based on a code change. I think we can live with this for now, but as we look into implementing auto-rfr for concurrent/python then we need to be aware.